lib/zipkin-tracer/rack/zipkin-tracer.rb

```diff
@@ -53,7 +53,7 @@ def annotate_plugin(env, status, response_headers, response_body)
 
     def trace!(span, zipkin_env, &block)
       # if called by a service, the caller already added the information
-      trace_request_information(span, zipkin_env.env) unless zipkin_env.called_with_zipkin_headers?
```

Comment:
in a shared span (which is the current model), this will duplicate the entry (once for the client and again for the server). For many, making this change will clutter the span view and also add data with no value unless there's some proxy going on. I'm working in the java (Brave) codebase to make something like this opt-in vs required for all users.

when/if we move to a single-host model, I'd agree with this change regardless, but for now I do think it will clutter most people's span detail screen.
https://github.com/openzipkin/zipkin/issues/939#issuecomment-264076909